roles/docker-garbage-collection/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# file path and name variables
+garbage_home: "/apis/docker_garbage_collection"
```

Comment:
Since we are not only running this role against API VMs, I think `/apis` prefix is not accurate here. Probably using `/docker` instead?